unmaven-project.sh

```diff
@@ -29,9 +29,9 @@ BUILDXML=build.xml
 
 download_jars() {
     rm -rf $WEBINFLIB $LIBSRC $TESTLIB
-    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom
+    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=runtime -DexcludeScope=provided -DexcludeTypes=pom
     mvn dependency:copy-dependencies -DoutputDirectory=$LIBSRC -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom -Dclassifier=sources
-    mvn dependency:copy-dependencies -DoutputDirectory=$TESTLIB -DexcludeScope=compile -DexcludeTypes=pom
+    mvn dependency:copy-dependencies -DoutputDirectory=$TESTLIB -DexcludeScope=runtime -DexcludeTypes=pom
     mvn dependency:copy-dependencies -DoutputDirectory=$LIBSRC -DexcludeScope=compile -DexcludeTypes=pom -Dclassifier=sources
```

Comment:
- [ ] This line gets source codes of the library stored in `$TESTLIB`.
You need to fix this like the above line.

Or you can put the copy to `$LIBSRC` into one line.

|  | compile | provoded | runtime | test | scope options |
|-|:-:|:-:|:-:|:-:|-|
| $WEBINFLIB | Y | N | Y | N | includeScope=runtime |
| $TESTLIB | N | Y | N | Y | excludeScope=runtime |
| $LIBSRC | Y | Y | Y | Y |  |

Reply:
remove option of `excludeScope`
a363cf6ec8af17bcbf148d397248b43dab6bfff9

Reply:
You got the same Jar twice due to your wrong fix.

Reply:
Removed duplicate command.
c119c1294bbf376a3a1f375d7d730dd8e47e203f